collectors/ebpf.plugin/ebpf.c

```diff
@@ -343,22 +343,25 @@ void write_io_chart(char *chart, char *family, char *dwrite, long long vwrite, c
 /**
  * Write chart cmd on standard output
  *
- * @param type      the chart type
- * @param id        the chart id
- * @param title     the chart title
- * @param units     the units label
- * @param family    the group name used to attach the chart on dashaboard
- * @param charttype the chart type
- * @param order     the chart order
+ * @param type      chart type
+ * @param id        chart id
+ * @param title     chart title
+ * @param units     units label
+ * @param family    group name used to attach the chart on dashaboard
+ * @param charttype chart type
+ * @param context   chart context
+ * @param order     chart order
  */
-void ebpf_write_chart_cmd(char *type, char *id, char *title, char *units, char *family, char *charttype, int order)
+void ebpf_write_chart_cmd(char *type, char *id, char *title, char *units, char *family,
+                          char *charttype, char *context, int order)
 {
-    printf("CHART %s.%s '' '%s' '%s' '%s' '' %s %d %d\n",
+    printf("CHART %s.%s '' '%s' '%s' '%s' '%s' '%s' %d %d\n",
            type,
            id,
            title,
            units,
            family,
+           context,
```

Comment:
should it be `(context ? context : "")` ?

Reply:
I did not use this, because when a pointer with NULL value was given to Netdata, it could interpret what was given and the plugin runs without any error, so I considered this not necessary.

Reply:
`it could interpret` you mean `printf` chagnes that to `"(null)"`? If so can we rely on this under all conditions?

Reply:
I took a deeper look in the case now, and I observed that we are dealing with `"(null)"` only for `name` on `plugins.d`, initially I assumed that this was present for all fields, because I knew this was happened for `names`. Thanks to take a look on this, the latest commit is fixing.
Now I will extend a discussion, should we also do a special work for variables like `context`? Considering that by default Netdata makes a `context` when is not given, I think we are missing this.